pkg/templates/replicas/template.go

```diff
@@ -0,0 +1,48 @@
+package replicas
+
+import (
+	"fmt"
+
+	"golang.stackrox.io/kube-linter/internal/stringutils"
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/extract"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/replicas/internal/params"
+)
+
+const (
+	templateKey = "minimum-replicas"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "Minimum recommended replicas not met",
```

Comment:
The human name is typically not a sentence
```suggestion
		HumanName:   "Minimum replicas",
```

Reply:
Fixed